hibernate-core/src/test/java/org/hibernate/jpa/test/transaction/ClosedEntityManagerWithJpaTransactionComplianceTest.java

```diff
@@ -0,0 +1,129 @@
+/*
+ * Hibernate, Relational Persistence for Idiomatic Java
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later.
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.jpa.test.transaction;
+
+import java.util.Map;
+import javax.persistence.EntityManager;
+import javax.persistence.EntityTransaction;
+
+import org.hibernate.cfg.AvailableSettings;
+import org.hibernate.jpa.test.BaseEntityManagerFunctionalTestCase;
+
+import org.junit.Test;
+
+import org.hamcrest.CoreMatchers;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertThat;
+
+/**
+ * @author Andrea Boriero
+ */
+public class ClosedEntityManagerWithJpaTransactionComplianceTest extends BaseEntityManagerFunctionalTestCase {
+
+	@Override
+	protected void addConfigOptions(Map options) {
+		options.put( AvailableSettings.JPA_TRANSACTION_COMPLIANCE, "true" );
+	}
+
+	@Test
+	public void getTransactionOnClosedEntityManagerShouldReturnNotNUll() {
+		EntityManager em = createEntityManager();
+		em.close();
+		assertThat( em.getTransaction(), CoreMatchers.notNullValue() );
+	}
+
+	@Test
+	public void getTransactionOnClosedEntityManagerCallIsActive() {
```

Comment:
Method naming again.. we are expecting *not* null.

Reply:
Also, in the interest of shortening names... not sure the `getTransactionOnClosedEntityManager...` "prefix" is needed for the test names when they are part of a class named `ClosedEntityManagerWithJpaTransactionComplianceTest`.

Same for the methods on `ClosedEntityManagerWithoutJpaTransactionComplianceTest` and `ClosedEntityManagerWithoutJpaTransactionComplianceTest`